src/components/LanguageSelection/LanguageSelection.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { translate } from 'react-i18next'
+
+import style from './LanguageSelection.scss'
```

Comment:
I usually call the imported CSS modules `styles`, not `style`. Just for consistency. :)